features/support/send_sigint_during_bisect.rb

```diff
@@ -5,8 +5,8 @@ module RSpec::Core::Formatters
   BisectProgressFormatter = Class.new(remove_const :BisectProgressFormatter) do
     RSpec::Core::Formatters.register self
 
-    def bisect_round_finished(notification)
-      return super unless notification.round == 3
+    def bisect_round_started(notification)
```

Comment:
It looks like you've removed `bisect_round_finished` but kept `bisect_round_started`.  Seems a little odd that they aren't paired.  Not sure if there's anything to do about that though...